src/App/Command/Git/CloneCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\YiiDevTool\App\Command\Git;
+
+use Symfony\Component\Process\Process;
+use Yiisoft\YiiDevTool\App\Component\Console\PackageCommand;
+use Yiisoft\YiiDevTool\App\Component\Package\Package;
+use Yiisoft\YiiDevTool\App\PackageService;
+
+final class CloneCommand extends PackageCommand
+{
+    protected static $defaultName = 'git/clone';
+    protected static $defaultDescription = 'Clone packages repositories';
```

Comment:
```suggestion
    protected static $defaultDescription = 'Clone package repositories';
```

Reply:
WDYT about "Git repository clone command"?

Reply:
Replace on "Clone of package repositories" :)